src/Prismic/Exception/RequestFailureException.php

```diff
@@ -18,23 +20,54 @@ class RequestFailureException extends RuntimeException
 
     /**
      * Factory to return a Prismic Exception wrapping a Guzzle Exception
+     *
+     * @param GuzzleException $e
+     *
+     * @return self
```

Comment:
This information is already present in the method signature

Reply:
I think it's all-right to keep for some editors